readme.md

```diff
@@ -40,13 +40,20 @@ Pact.crypto.toTweetNaclSecretKey(keyPair) -> <Uint8Array>
 ### Language Expression Construction
 
 A helper function for constructing native Pact commands.
-
-```
-Pact.lang.mkExp(<string>, *args) -> <string>
+- mkExp takes in Pact function and its arguments and returns a Pact expression.
```

Comment:
backticks for function names (`mkExp`,`mkMeta`, etc)

Reply:
Changed